.github/workflows/playground-check.yaml

```diff
@@ -0,0 +1,68 @@
+name: Playground operation check
+
+on:
+  pull_request:
+  schedule:
+    # Run this job once per day
+    - cron: "0 0 * * *"
+
+jobs:
+  playground-check:
+    runs-on: ubuntu-latest
+    env:
+      FORK_URL_MAINNET: ${{ secrets.FORK_URL_MAINNET }}
+    steps:
+      - name: Free Disk Space (Ubuntu)
+        uses: jlumbroso/free-disk-space@main
+        with:
+          tool-cache: false
+          android: true
+          dotnet: true
+          haskell: true
+          large-packages: false
+          docker-images: false
+          swap-storage: false
```

Comment:
Why do we need this step and especially the enabled `android`, dotnet and `haskell` flags?

Reply:
Without freeing some space on github runner we was running out of disk space, here is example run:
https://github.com/cowprotocol/services/actions/runs/11515686142/attempts/1
I added some logging of free space on github runner, and after start we have 21 GB available, but building docker containers require 22 GB (after freeing space we know that). 
I just left `android`, `dotnet` and `haskell` as `true` because it is enough and execution time of that step is short, comparing to run with other steps enabled.